agreementbot/agreementbot.go

```diff
@@ -1142,6 +1179,11 @@ func (w *AgreementBotWorker) syncOnInit() error {
 				}
 			}
 
+			glog.V(3).Infof(AWlogString(fmt.Sprintf("discovered changed/deleted business policies: %v", bPolicyMessageMap)))
```

Comment:
I would not say "changed" policies in the log message.... Deleted is ok but maybe "policies that might have changed"